phoonnx/cli.py

```
@@ -110,8 +111,40 @@ def list_voices(lang, verbose):
+    """
+    Lists all voice IDs available from upstream sources (Piper, Mimic3, etc.),
+    even if they are not yet in the local cache. (Requires network connection)
+    """
+    manager = TTSModelManager()
+    
+    click.echo("Fetching all available voice IDs from upstream sources (Piper, Mimic3, OVOS, etc.)...")
+    
+    try:
+        available_voices = manager.get_all_available_voice_ids()
+    except requests.exceptions.RequestException as e:
+        click.echo(f"\nError: Could not fetch available voice lists due to a network or connection error.", err=True)
```

Ruff (0.14.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)